SimulationAgent/appsettings.ini

```diff
@@ -5,12 +5,37 @@ iothub_connstring = "${PCS_IOTHUB_CONNSTRING}"
 device_models_folder = ./data/devicemodels/
 device_models_scripts_folder = ./data/devicemodels/scripts/
 
+[DeviceSimulationService:RateLimits]
+# S3:  5000/min/unit (= 83.3/sec/unit)
+# S2:   100/min/unit (= 1.67/sec/unit)
+# S1:   100/min/unit (= 1.67/sec/unit)
+# F1:   100/min      (= 1.67/sec)
+registry_operations_per_minute = 100
```

Comment:
I'd like to limit to 1/2 or 3/4 the available limits to allow for clients outside the simulator leeway instead of running the simulator @ capacity.
e.g. registry_operations_per_minute = 50 or 75

Reply:
sure, let's recap all the limits in a separate task and leave these here for this PR?

Reply:
https://github.com/Azure/device-simulation-dotnet/issues/93

By the way the values in the repository might differ from what we want to use from deployment to deployment, and for other solutions